demo/src/main/resources/templates/event-type/event-type-form.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Add Event</title>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
+    <link href="../static/styles.css" rel="stylesheet">
+</head>
+<body>
+<header th:insert="~{header.html}"></header>
+<div class="container mt-5">
+    <div class="row justify-content-center">
+        <div class="col-md-6">
+            <div class="card">
+                <div class="card-header">
+                    <h3 class="text-center">Add Event</h3>
+                </div>
+                <div class="card-body">
+                    <form action="#" th:action="@{/event-type/submit}" th:object="${eventTypeDTO}" method="post">
+                        <div class="form-group">
+                            Event type:
+                            <input type="text" th:field="*{typeName}">
+                            <span th:if="${#fields.hasErrors('typeName')}" th:errors="*{typeName}"></span>
+                        </div>
+                        <div>
+                            <input type="hidden" name="id" th:value="${id}"/>
```

Comment:
I don't think we need to pass id as a hidden type param in this form, if we send and eventType to the backend without the id and use eventTypeRepository.save(NewEventType); the id would be automaticaly set in the db. We only need to pass the id as a hidden type param when we are updating an eventType, in that scenario we have the id to make sure that we aren't adding a new eventType in the db but changing the fields of an already existing eventType.